pkgs/desktops/gnome/artwork/firewatch/default.nix

```diff
@@ -0,0 +1,230 @@
+        <name xml:lang="he">&#x5E8;&#x5E7;&#x5E2; &#x5D1;&#x5E8;&#x5E8;&#x5EA; &#x5DE;&#x5D7;&#x5D3;&#x5DC;</name>
+        <name xml:lang="gu">&#xAAE;&#xAC2;&#xAB3;&#xAAD;&#xAC2;&#xAA4; &#xAAA;&#xABE;&#xAB6;&#xACD;&#xAB0;&#xACD;&#xAB5;&#xAAD;&#xABE;&#xA97;</name>
+        <name xml:lang="gl">Fondo de escritorio predeterminado</name>
+        <name xml:lang="gd">An c&#xF9;laibh t&#xF9;sail</name>
+        <name xml:lang="ga">C&#xFA;lra R&#xE9;amhshocraithe</name>
+        <name xml:lang="fur">Sfont predefin&#xEE;t</name>
+        <name xml:lang="fr">Arri&#xE8;re-plan par d&#xE9;faut</name>
+        <name xml:lang="fi">Oletustaustakuva</name>
+        <name xml:lang="fa">&#x67E;&#x633;&#x200C;&#x632;&#x645;&#x6CC;&#x646;&#x647; &#x67E;&#x6CC;&#x634;&#x200C;&#x6AF;&#x632;&#x6CC;&#x62F;&#x647;</name>
+        <name xml:lang="eu">Atzeko plano lehenetsia</name>
+        <name xml:lang="et">Vaikimisi taust</name>
+        <name xml:lang="es">Fondo predeterminado</name>
+        <name xml:lang="eo">Implicita fono</name>
+        <name xml:lang="en_GB">Default Background</name>
+        <name xml:lang="el">&#x3A0;&#x3C1;&#x3BF;&#x3B5;&#x3C0;&#x3B9;&#x3BB;&#x3B5;&#x3B3;&#x3BC;&#x3AD;&#x3BD;&#x3BF; &#x3C0;&#x3B1;&#x3C1;&#x3B1;&#x3C3;&#x3BA;&#x3AE;&#x3BD;&#x3B9;&#x3BF;</name>
+        <name xml:lang="de">Standard-Hintergrund</name>
+        <name xml:lang="da">Standardbaggrund</name>
+        <name xml:lang="cs">v&#xFD;choz&#xED; pozad&#xED;</name>
+        <name xml:lang="ca@valencia">Fons d'escriptori per defecte</name>
+        <name xml:lang="ca">Fons d'escriptori per defecte</name>
+        <name xml:lang="bs">Podrazumijevana pozadina radne povr&#x161;ine</name>
+        <name xml:lang="bg">&#x421;&#x442;&#x430;&#x43D;&#x434;&#x430;&#x440;&#x442;&#x43D;&#x438; &#x444;&#x43E;&#x43D;&#x43E;&#x432;&#x435;</name>
+        <name xml:lang="be">&#x41F;&#x440;&#x430;&#x434;&#x432;&#x44B;&#x437;&#x43D;&#x430;&#x447;&#x430;&#x43D;&#x44B; &#x444;&#x43E;&#x43D;</name>
+        <name xml:lang="as">&#x985;&#x9AC;&#x9BF;&#x995;&#x9B2;&#x9CD;&#x9AA;&#x9BF;&#x9A4; &#x9AA;&#x99F;&#x9AD;&#x9C2;&#x9AE;&#x9BF;</name>
+        <name xml:lang="ar">&#x627;&#x644;&#x62E;&#x644;&#x641;&#x64A;&#x629; &#x627;&#x644;&#x627;&#x641;&#x62A;&#x631;&#x627;&#x636;&#x64A;&#x629;</name>
+        <name xml:lang="an">Fondo predeterminau</name>
+        <name xml:lang="af">Verstek agtergrond</name>
+        <filename>$out/share/backgrounds/animated/Firewatch.xml</filename>
+        <options>zoom</options>
+        <shade_type>solid</shade_type>
+        <pcolor>#3465a4</pcolor>
+        <scolor>#000000</scolor>
+      </wallpaper>
+    </wallpapers>
+
+
+    EOF
+        runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Lakeside Gnome dynamix wallpaper with more frames.";
+    homepage = "https://github.com/manishprivet/dynamic-gnome-wallpapers";
+    license = with licenses; [ mit ];
```

Comment:
The repo claims MIT but there does not appear to be any license provided for the images themselves so it should probably be `unfree`.